apps/website-25/src/components/courses/exercises/FreeTextResponse.tsx

```diff
@@ -1,36 +1,55 @@
 import clsx from 'clsx';
```

Comment:
This is so much cleaner without having to have all the networking logic here 🎉 

Excited that this also makes it easier to show in storybook and write tests for. Plus building new types of exercises should be easier.